packages/fx-core/src/plugins/resource/bot/languageStrategy.ts

```diff
@@ -66,11 +66,6 @@ export class LanguageStrategy {
       appSettings = [];
     }
 
-    appSettings.push({
-      name: "SCM_DO_BUILD_DURING_DEPLOYMENT",
-      value: "true",
-    });
-
     appSettings.push({
       name: "WEBSITE_NODE_DEFAULT_VERSION",
```

Comment:
What if user local node version is not 12 but 14? Will it lead to some compatible issue from remote to local? In Function plugin, it will detect user local machine node version and create the same node version in Azure. If this node version is not supported in Azure, we will use 12 as default value. Can you help sync with Heng to check if we need to apply the same to Bot plugin?